stdlib/camlinternalFormat.ml

```diff
@@ -2055,15 +2055,15 @@ let fmt_ebb_of_string ?legacy_behavior str =
     invalid_format_message str_ind
       "non-zero widths are unsupported for %c conversions"
   in
-  (* Raise Failure with a friendly error message about an option dependencie
+  (* Raise a Failure with a friendly error message about an option dependency
```

Comment:
Either `Raise [Failure]` or `Raise a Failure exception` (and later). Personally, for documenting exception behaviour, I prefer `Raise [Failure]`.

I'm not sure about the context - should it be option or optional?